src/sass/components/_tour.scss

```diff
@@ -75,3 +115,7 @@
   text-align: left;
   white-space: initial;
 }
+
+.inline-container {
+  display: inline-block;
+}
```

Comment:
@dvdherron I'm not sure what happened to my comment on this one so forgive me if it is duplicate, but I think this could move to one of the display (data-helpers or containers) scss files in the patterns folder. It feels like a more generic pattern that can be used elsewhere even though we are currently only using it on tour items at this time.

Reply:
👍🏽  moved to `_containers`